examples/quote_of_day.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const libingester = require('libingester');
+const mustache = require('mustache');
+const rp = require('request-promise');
+const template = require('./quote_of_day_template');
+const url = require('url');
+const home_page = 'http://www.thefreedictionary.com/_/archive.htm'; // Home section
+
+function ingest_article_profile(hatch, uri) {
+    return libingester.util.fetch_html(uri).then(($profile) => {
+        const base_uri = libingester.util.get_doc_base_uri($profile, uri);
+
+        const asset = new libingester.NewsArticle();
+        asset.set_canonical_uri(uri);
+
+        // Use 'url' module to pull out query string date object
+        const parts = url.parse(uri, true);
+        const modified_date = new Date(Date.parse(parts.query.d));
+        asset.set_last_modified_date(modified_date);
+
+        // Set date and title
+        const date = parts.query.d;
```

Comment:
Move this up so that you can use this variable in `Date(Date.parse(parts.query.d));` statement earlier. Also, `date` as a name isn't descriptive enough.